gemsupervisor/src/common/GEMSupervisor.cc

```diff
@@ -384,8 +384,6 @@ void gem::supervisor::GEMSupervisor::configureAction()
     m_globalState.update();
   }
 
-  pausePhaseMonitoring();
```

Comment:
What was the reason to pause phase monitoring an was this change tested?

Reply:
The `pause`/`resume` was implemented when we didn't have the `memhub` protection against potentially colliding transactions.